packages/faucet/src/api/webserver.ts

```diff
@@ -58,13 +59,22 @@ export class Webserver {
           // context.request.body is set by the bodyParser() plugin
           const requestBody = context.request.body;
           const creditBody = RequestParser.parseCreditBody(requestBody);
-
           const { address, denom } = creditBody;
 
           if (!isValidAddress(address, constants.addressPrefix)) {
             throw new HttpError(400, "Address is not in the expected format for this chain.");
           }
 
+          const entry = this.addressCounter.get(address);
+          if (entry !== undefined) {
+            if (entry.getDate() + 24 * 3600 > Date.now()) {
```

Comment:
>  The getDate() method returns the day of the month for the specified date according to local time. 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getDate

Seems like this is not what we want here

Reply:
I'm stupid. Got confused. It's `getTime()` ofc